packages/theming/apple-theme/src/appleColors.types.macos.ts

```diff
@@ -1,131 +1,130 @@
 import { ColorValue } from 'react-native';
-import { PlatformColor } from 'react-native-macos';
```

Comment:
Yayy

Reply:
I can't believe this code made it through so far and was obviously a typo :D